test/utils/utils.cpp

```diff
@@ -13,9 +13,9 @@ evmc_revision to_rev(std::string_view s)
         return EVMC_FRONTIER;
     if (s == "Homestead")
         return EVMC_HOMESTEAD;
-    if (s == "EIP150")
+    if (s == "Tangerine Whistle" || s == "EIP150")
```

Comment:
This may not be the best idea because e.g. geth doesn't understand "Tangerine Whistle".

Reply:
I changed this to export as `EIP150` and `EIP158`... We still accept full revision names.